### PR TITLE
Bugfix to avoid newline character \n in customFormSubmit URL

### PR DESCRIPTION
**Pull request by eneiasramos**
### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)
 - [ ] Add yourself to contributors list

### CODE REVIEW

**Comment by sreichel** [comment]
Thanks. Can you please add some/most relevant information? Reviews are easier when you know the problem from reading and see how its fixed :) _(imho)_

File: app/design/frontend/rwd/default/template/email/catalog/product/list.phtml [hunk 1]
```diff
@@ -84,13 +84,11 @@ $_params = $this->escapeHtml(json_encode(['form_key' => $this->getFormKey()]));
...
                                         <p class="action">
+                                            <?php $_cartUrl = $this->getAddToCartUrlCustom($_product, [], false); ?>
                                             <button type="button"
                                                     title="<?php echo $this->quoteEscape($this->__('Add to Cart')) ?>"
                                                     class="button btn-cart"
-                                                    onclick="customFormSubmit(
-                                                            '<?php echo $this->getAddToCartUrlCustom($_product, [], false) ?>',
-                                                            '<?php echo $_params ?>',
-                                                            'post')">
+                                                    onclick="customFormSubmit('<?php echo $_cartUrl ?>', '<?php echo $_params ?>', 'post')">
```
Comment (fballiano) [inline app/design/frontend/rwd/default/template/email/catalog/product/list.phtml:91]: isn't this code exactly the same as before?

**Comment by eneiasramos** [comment]
@sreichel I was inspecting an issue with the onclick event and a 3rd party plugin, then I found this:

a javascript call with \n between the parameters.

image

File: app/design/frontend/rwd/default/template/email/catalog/product/list.phtml (hunk 1 above)
Comment (eneiasramos) [reply app/design/frontend/rwd/default/template/email/catalog/product/list.phtml:91]: @fballiano I looked at the history of this file and didn't find this code.

**Comment by sreichel** [comment]
I agree with this change to put this in one line, but i'd use .... without `$_cartUrl`

```
onclick="customFormSubmit('<?php echo $this->getAddToCartUrlCustom($_product, [], false) ?>', '<?php echo $_params ?>', 'post')">
```

**Comment by eneiasramos** [comment]
@sreichel I understand your point of view but I followed the same pattern as the lines below:

https://github.com/OpenMage/magento-lts/blob/4db847a8ff61410225f8a7b812ccf8e74e2415a0/app/design/frontend/rwd/default/template/catalog/product/list.phtml#L104

https://github.com/OpenMage/magento-lts/blob/4db847a8ff61410225f8a7b812ccf8e74e2415a0/app/design/frontend/rwd/default/template/catalog/product/list.phtml#L115

**Review by sreichel — APPROVED** [review]

**Comment by sreichel** [comment]
> I understand your point of view but I followed the same pattern as the lines below:

Ah okay. Next time, please put such info into PR description ... :)

**Comment by eneiasramos** [comment]
> > I understand your point of view but I followed the same pattern as the lines below:
> 
> Ah okay. Next time, please put such info into PR description ... :)

@sreichel perfect my dear! I will do this :)

Thank you! o/

**Pull request merged**